api/v1alpha1/multiclusterengine_types.go

```diff
@@ -34,6 +34,9 @@ type MultiClusterEngineSpec struct {
 
 	// Tolerations causes all components to tolerate any taints.
 	Tolerations []corev1.Toleration `json:"tolerations,omitempty"`
+
+	// Location where MCE resources will be placed
+	TargetNamespace string `json:"targetNamespace,omitempty"`
```

Comment:
Id recommend adding API markers for this field to prettify/simplify its view in the UI. 
See - 
https://sdk.operatorframework.io/docs/building-operators/golang/references/markers/#usage
And 
https://github.com/openshift/console/blob/master/frontend/packages/operator-lifecycle-manager/src/components/descriptors/reference/reference.md#14-selector